libmscore/sym.cpp

```diff
@@ -2921,11 +2921,11 @@ const std::array<const char*, int(SymId::lastSym)+1> Sym::symUserNames = { {
       "23 small diesis down",
       "23 small diesis up, (23S)",
       QT_TRANSLATE_NOOP("symUserNames", "25 small diesis down, 2° down [53 EDO]"),
-      QT_TRANSLATE_NOOP("symUserNames", "25 small diesis up, (25S, ~5:13S, ~37S, 5C plus 5C), 2° up [53 EDO]"),
+      QT_TRANSLATE_NOOP("symUserNames", "25 small diesis up, (25S, ~5:13S, ~37S, 5C plus 5C), 2° up [53 EDO]"),
```

Comment:
What happened here?

Reply:
probably an unbreakable space

Reply:
Yes, indeed. fonts/smufl/glyphnames.json has it as `\u00a0`, seems libmscore/sym.coo had a plain blank before.

Reply:
I hole it doesn't screw up translations, esp. the 2nd of them which is followed by something that can get interpreted as a a (hex) digit.
Guess we'll know soon enough (once translations are enabled again), and can react on it if and when problems arise.
But this is something to keep an eye on.

Reply:
Honestly, I have no idea where they came from